src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const TodoForm = props => {
+    return (
+        <form onSubmit={props.handleSubmit}>
+        <input
+        type="text"
+        value={props.task}
+        name="task"
+        onChange={props.handleChange}
+        placeholder="...todo"
+        />
+<button onClick={props.handleSubmit}>Add Todo</button>
+<button onClick={props.handleSubmit}>Clear Completed</button>
+        </form>
```

Comment:
once you get your clear method working you will want to add that to the second button here instead of the handleSubmit unless you are going to handle the logic conditionally inside the handleSubmit method